core/modules/layout/plugins/access/http_response_code_layout_access.inc

```diff
@@ -0,0 +1,98 @@
+<?php
+/**
+ * @file
+ * Class for loading, modifying, and saving a layout access rule (condition).
```

Comment:
Most of our other class files don't have both a file docblock and a class docblock. Should be consistent with the rest of core, or make the code pass code sniffer?